test/test-mda-models/test-data-delivery-spark-model/src/test/java/com/boozallen/aiops/mda/pattern/SparkSchemaTest.java

```
@@ -169,6 +191,16 @@ public void sparkSchemaValidationIsPerformedOnThePersonWithMToOneRelationDataSet
         }
     }
 
+    @When("spark schema validation is performed on the \"PersonWithOneToMRelation\" dataSet")
+    public void sparkSchemaValidationIsPerformedOnThePersonWithOneToMRelationDataSet() {
+        try {
+            this.validatedDataSet =
+                    this.personWithOneToMRelationSchema.validateDataFrame(this.personWithOneToMRelationDataSet);
+        }catch (Exception e) {
+            this.exception = e;
```

a: If we are no longer using this.exception in the validation then we should remove the catch and let it fail naturally